adit/core/processors.py

```diff
@@ -313,7 +313,15 @@ def callback(ds: Dataset | None) -> None:
 
             final_folder: Path
             if settings.CREATE_SERIES_SUB_FOLDERS:
-                series_folder_name = sanitize_filename(f"{ds.SeriesNumber}-{ds.SeriesDescription}")
+                series_number = ds.get("SeriesNumber", "")
+                if series_number == "":
+                    series_folder_name = ds.SeriesInstanceUID
+                else:
+                    series_description = ds.get("SeriesDescription", "")
```

Comment:
Why not just `ds.get("SeriesDescription", "Undefined")` when you are doing that anyway in the following line?! I also vote for "unknown".